scripts/popup.js

```diff
@@ -300,6 +300,27 @@ function settings() {
   $('#setting-page').show()
 }
 
+function share(){
+  const webstoreURL = 'https://chrome.google.com/webstore/detail/wayback-machine/fpnmgdkabkmnadcjpehmlllkndpkmiak'
+  function showToast(toastMsg){
```

Comment:
Why you have created a function here?

Reply:
To reuse the code to show the toast.

Reply:
Define it out side the share function then !Thank you